app/code/Magento/Catalog/Helper/Output.php

```diff
@@ -94,7 +102,7 @@ protected function _getTemplateProcessor()
      */
     public function addHandler($method, $handler)
     {
-        if (!is_object($handler)) {
+        if (!\is_object($handler)) {
```

Comment:
Please import it similar to `strtolower`. 

After changes are applied and all builds are green, please squash them into a single commit so that we have perfectly clean history 😉